### ISSUE TITLE
Remove duplicate timeout setting

### ISSUE THREAD

**Issue by rrrooommmaaa** [FlowCrypt/flowcrypt-browser#5040]
We have `TIMEOUT_OVERALL` setting in `test/source/test.ts`
and we also pass timeout as CLI argument in `package.json` scripts e.g. `npx ava --timeout=30m --verbose build/test/test/source/test.js`

Isn't this excessive?

**Comment by tomholub** [comment]
I think that the internally set `TIMEOUT_OVERALL` is meant to cause a softer fail then the external `--timeout`. To that end, the external one should be set to a higher value than the internal one.

I don't know what the difference is in practical terms, maybe more debug info is still printed when the internal timeout is reached.

**Comment by tomholub** [comment]
If it bothers you that we have 3 timeouts, then you could remove the `--timeout` in particular as it seems that one brings the least value.

**Comment by rrrooommmaaa** [comment]
Yes, it'd be helpful -- less time will be spent next time for reconfiguring the tests

**Comment by sosnovsky** [comment]
Hi @ioanatflowcrypt, can you please check if this issue is relevant or can be closed, thanks!

**Comment by ioanatflowcrypt** [comment]
Ok
